docker/compose.yml

```diff
@@ -511,10 +509,9 @@ services:
       - ./config/zkbesu-shomei/zkbesu-config.toml:/var/lib/besu/zkbesu-config.toml:ro
       - ./config/zkbesu-shomei/log4j.xml:/var/lib/besu/log4j.xml:ro
       - ./config/linea-local-dev-genesis-PoA-besu.json/:/var/lib/besu/genesis.json:ro
-      - ../tmp/zkbesu-shomei/plugins:/opt/besu/plugins/
 
   shomei:
-    image: consensys/linea-shomei:2.2.0
+    image: consensys/linea-shomei:2.3.0
```

Comment:
Don't you need to use `linea-besu-package` here?

Reply:
No, Shomei is a different service decoupled from Besu.

Reply:
Right, I can see you made the change above